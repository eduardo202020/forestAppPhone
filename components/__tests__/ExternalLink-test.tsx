import React from 'react';
import renderer from 'react-test-renderer';
import { ExternalLink } from '../ExternalLink';

describe('ExternalLink', () => {
  it('renderiza correctamente el link', () => {
    const tree = renderer
      .create(
        <ExternalLink href="https://example.com">Ir a ejemplo</ExternalLink>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
