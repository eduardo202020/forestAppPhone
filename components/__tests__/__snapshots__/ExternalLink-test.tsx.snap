// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExternalLink renderiza correctamente el link 1`] = `
<Text
  href="https://example.com"
  onPress={[Function]}
  role="link"
>
  Ir a ejemplo
</Text>
`;
